baus/variables.py

```diff
@@ -149,26 +149,6 @@ def vacant_res_units(buildings, households):
 @orca.column('buildings', cache=True)
 def sqft_per_job(buildings, building_sqft_per_job, sqft_per_job_adjusters, telecommute_sqft_per_job_adjusters, taz_geography, base_year, year, run_setup):
 
-    # get building types list
-    mapping = orca.get_injectable("mapping")
```

Comment:
This is actually moved down, not deleted - it is moved to the block where it will necessarily be used - no reason to run if the relevant flag is not set.